### PR TITLE
279765 user payment request journey

### PR DESCRIPTION
**Pull request by JasmeetKaur-Defra**

### CODE REVIEW

File: EST.MIT.Web/Pages/invoice/ReadonlyInvoiceSummary/ReadonlyInvoiceSummary.razor [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @page "/invoice/view/{id}"
-@page "/invoice/details/{scheme}/{id}/{approval:bool?}"
+@page "/invoice/details/{scheme}/{id}/{paymentRequestId}/{approval:bool?}"
```
Comment (andygibbonsdefra) [inline EST.MIT.Web/Pages/invoice/ReadonlyInvoiceSummary/ReadonlyInvoiceSummary.razor:2]: Should this be saved as a completely new ReadOnlyPaymentRequestSummary page? (The original page was for the whole Invoice, which I think we still need.)

File: EST.MIT.Web/Pages/invoice/Summary/Summary.razor [hunk 2]
```diff
@@ -4,40 +4,77 @@
...
-		@if(!invoice.PaymentRequests.Any())
-		{
+            <div id="no-payment-request-warning" class="govuk-form-group--error">
+                <h1 class="govuk-heading-l">This invoice currently has no payment requests</h1>
+                <div class="govuk-hint">You can select Add Payment Request to include one with this invoice.</div>
+            </div>
+        }
+        else
+        {
+            @if (invoice.Status == "new")
```
Comment (andygibbonsdefra) [inline EST.MIT.Web/Pages/invoice/Summary/Summary.razor:66]: Invoice Status check should be done against the status enum.

Comment (andygibbonsdefra) [reply EST.MIT.Web/Pages/invoice/Summary/Summary.razor:66]: Can you check to see if there are other places the status is hard coded?

Comment (JasmeetKaur-Defra) [reply EST.MIT.Web/Pages/invoice/Summary/Summary.razor]: Resolved

File: EST.MIT.Web/Pages/invoice/ReadonlyInvoiceSummary/ReadonlyInvoiceSummary.razor (hunk 1 above)
Comment (JasmeetKaur-Defra) [reply EST.MIT.Web/Pages/invoice/ReadonlyInvoiceSummary/ReadonlyInvoiceSummary.razor]: Created a separate page for read only payment request

**Review by andygibbonsdefra — APPROVED** [review]

**Pull request merged**